src/write.ts

```diff
@@ -537,3 +538,75 @@ export async function writeBenchmark(bench: Benchmark, config: Config) {
         await handleAlert(name, bench, prevBench, config);
     }
 }
+
+export async function writeSummary(bench: Benchmark, config: Config): Promise<void> {
+    const { name, externalDataJsonPath } = config;
+    const prevBench = externalDataJsonPath
+        ? await writeBenchmarkToExternalJson(bench, externalDataJsonPath, config)
+        : await writeBenchmarkToGitHubPages(bench, config);
+
+    if (prevBench === null) {
+        core.debug('Alert check was skipped because previous benchmark result was not found');
+        return;
+    }
+
+    const headers = [
+        {
+            data: 'Benchmark Suite',
+            header: true,
+        },
+        {
+            data: `Current: "${bench.commit}"`,
+            header: true,
+        },
+        {
+            data: `Previous: "${prevBench.commit}"`,
+            header: true,
+        },
+        {
+            data: 'Ratio',
+            header: true,
+        },
+    ];
+    const rows: SummaryTableRow[] = bench.benches.map((bench) => {
+        const previousBench = prevBench.benches.find((pb) => pb.name === bench.name);
+
+        if (previousBench) {
+            const ratio = biggerIsBetter(config.tool)
+                ? previousBench.value / bench.value
+                : bench.value / previousBench.value;
+
+            return [
+                {
+                    data: bench.name,
+                },
+                {
+                    data: strVal(bench),
+                },
+                {
+                    data: strVal(previousBench),
+                },
+                {
+                    data: floatStr(ratio),
+                },
+            ];
+        }
+
+        return [
+            {
+                data: bench.name,
+            },
+            {
+                data: strVal(bench),
+            },
+            {
+                data: '-',
+            },
+            {
+                data: '-',
+            },
+        ];
+    });
+
+    core.summary.addHeading(`Benchmarks: ${name}`).addTable([headers, ...rows]);
```

Comment:
Note this is missing the call to `write()` to actually send the summary upstream to GitHub

```
core.summary.addHeading(`Benchmarks: ${name}`).addTable([headers, ...rows]).write();
```

Reply:
Thanks!